woocommerce-simple-registration.php

```diff
@@ -254,6 +254,10 @@ public function account_settings( $settings ) {
 				'id'            => 'woocommerce_simple_registration_register_page',
 				'default'       => '',
 				'type'          => 'single_select_page',
+				'args'          => array(
+					'option_none_value' => 0,
+					'show_option_none' => esc_html__( '&mdash; Select &mdash;', 'woocommerce-simple-registration' ),
```

Comment:
Let's do `Select a page...` to match the others: https://cloudup.com/cyo_yltIKPT